rj_gameplay/stp/utils/formations.py

```diff
@@ -0,0 +1,141 @@
+from typing import List
+
+import numpy as np
+import stp
+from stp.utils.constants import RobotConstants
+
+
+class Formations:
```

Comment:
this is written a bit like a static method, should have types of Formations subclass some abstract Formation class and pass that in instead